src/SSW.SophieBot/SSWSophieBot/language-generation/en-us/greeting.en-us.lg

```diff
@@ -0,0 +1,57 @@
+# GreetingHeroCard()
+- ```
+{
+    "text": "${SayHi()}",
+    "images": [
+        ${if(!user.welcomed, GreetingImage(), '')}
+    ],
+    "buttons": [
+        ${GreetingButtons()}
+    ]
+}
+```
+
+# SayHi()
+- IF: ${turn.signedInUser.givenName != null}
+    - ```
+    <b><font size=\"4\">Welcome to SSW SophieBot!</font></b><br/><br/>
+    <b>Hi ${turn.signedInUser.givenName}</b> 🖐<br/><br/>
+    ${SummaryText()}
+    ```
+- ELSE: 
+    - ```
+    <b><font size=\"4\">Welcome to SSW SophieBot!</font></b><br/><br/>
+    <b>Hi there</b> 🖐<br/>
+    ${SummaryText()}
+    ```
+
+# GreetingImage()
+- ```
+{
+    "url": "https://www.ssw.com.au/ssw/logo/sophiebot/Images/SophieBot_preview.png"
+}
+```
+
+# SummaryText()
+- I can help you search for people in your organization. Try by asking.
```

Comment:
Can we put the `Try by asking...` to the new line?
![image](https://user-images.githubusercontent.com/37203901/138012551-5f44c04a-ec01-4dcf-bece-16ad75171086.png)
**Figure: Text not well formatted**